services/threat-library-service/src/utils/cypher-generator.ts

```
@@ -0,0 +1,500 @@
+  // Build WHERE clause
+  const whereClause =
+    conditions.length > 0 ? `WHERE ${conditions.join(' AND ')}` : '';
+
+  // Build RETURN clause based on format
+  let returnClause: string;
+  switch (options.returnFormat) {
+    case 'count':
+      returnClause = 'RETURN count(*) AS matchCount';
+      break;
+    case 'paths':
+      const pathNodes = Array.from(nodeAliases.values()).join(', ');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)